build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id "com.github.johnrengelman.shadow" version "6.1.0" apply false
+}
+
+subprojects {
+    apply plugin: "java"
+    apply plugin: "maven-publish"
+
+    sourceCompatibility = 11
+    targetCompatibility = 11
+    version rootProject.file(".loomversion").text
+
+    ext {
+        mcVersion = version
+        jbsdiffVersion = "5df6d1a09b"
+    }
+
+    tasks.withType(JavaCompile) {
+        options.encoding = "UTF-8"
+    }
+
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+        maven { url "https://maven.fabricmc.net/" }
+        maven { url "https://jitpack.io/" }
+    }
```

Comment:
Order matters here, so I would prefer if `mavenLocal()` was first.

Why do we need the fabric one?

The jitpack.io one is already added in the launcher specific build file? So I think it can be omitted here as well?

Reply:
Done